pkg/sql/show_create_clauses.go

```diff
@@ -48,6 +48,8 @@ type comment struct {
 
 // selectComment retrieves all the comments pertaining to a table (comments on the table
 // itself but also column and index comments.)
+// TODO(chengxiong): consider plumbing the collection through here so that we
+// can just fetch comments from collection cache instead of firing extra query.
```

Comment:
An alternative approach would be to leverage a virtual table which internally uses the collection.